src/lib/InfoWindow.js

```diff
@@ -72,7 +72,7 @@ const publicMethodMap = {
 
 const controlledPropUpdaterMap = {
   children(infoWindow, children) {
-    render(Children.only(children), infoWindow.getContent());
+    renderSubtreeIntoContainer(this, Children.only(children), infoWindow.getContent());
```

Comment:
I'd like to keep it just `unstable_renderSubtreeIntoContainer` to indicate we're using unstable APIs.